<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="Create">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>351</width>
     <height>672</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>35</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>35</number>
    </property>
    <property name="bottomMargin">
     <number>98</number>
    </property>
    <property name="verticalSpacing">
     <number>40</number>
    </property>
    <item row="6" column="0">
     <widget class="QLineEdit" name="password_user">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Lösenord</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="first_name">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Förnamn</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="username">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>Användarnamn</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="last_name">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>Efternamn</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="email">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="confirm_password">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>Bekräfta Lösenord</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>40</number>
      </property>
      <item>
       <widget class="QPushButton" name="create">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-style: solid;
	border-color: #9999aa;
	border-radius: 5px;
	color: white;
	background-color: #3474eb;
	height: 40px
}

QPushButton:enabled {
	background-color: #2C65A4;
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: #0034AB;
	color: white;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
        </property>
        <property name="text">
         <string>Skapa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-style: solid;
	border-color: #9999aa;
	border-radius: 5px;
	color: white;
	background-color: #3474eb;
	height: 40px
}

QPushButton:enabled {
	background-color: #2C65A4;
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: #0034AB;
	color: white;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
        </property>
        <property name="text">
         <string>Tillbaka</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>400</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Bilder/Skärmavbild 2024-05-02 kl. 00.15.18.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
